homeassistant/components/xiaomi_tv/media_player.py

```diff
@@ -69,16 +91,42 @@ def state(self):
         """Return _state variable, containing the appropriate constant."""
         return self._state
 
+    def update(self):
+        """Update the device state."""
+        _LOGGER.debug("update: %s", self.entity_id)
+        if self._assume_state:
+            return self._state
```

Comment:
`update` method shouldn't return anything. We want to fetch state from the device and set the new state in the entity instance attributes.